components/admin/requests/additional-questions/Form.tsx

```diff
@@ -1,55 +1,106 @@
-import { FormControl, FormLabel, Stack, Radio, RadioGroup } from '@chakra-ui/react'; // Chakra UI
+import { Stack, Radio, Box } from '@chakra-ui/react'; // Chakra UI
+import RadioGroupField from '@components/form/RadioGroupField';
+import TextAreaField from '@components/form/TextAreaField';
+import {
+  AccessibleConvertedVanLoadingMethod,
+  RequiresWiderParkingSpaceReason,
+} from '@prisma/client';
 import { AdditionalInformationFormData } from '@tools/admin/requests/additional-questions'; // AdditionalQuestions type
 
 type AdditionalQuestionsFormProps = {
-  data: AdditionalInformationFormData;
-  onChange: (additionalQuestions: AdditionalInformationFormData) => void;
+  additionalInformation: AdditionalInformationFormData;
 };
 
-export default function AdditionalQuestionsForm({ data, onChange }: AdditionalQuestionsFormProps) {
+export default function AdditionalQuestionsForm({
+  additionalInformation,
+}: AdditionalQuestionsFormProps) {
   return (
     <>
-      <FormControl as="fieldset" isRequired>
-        <FormLabel>{'Is the applicant using an accessible converted van?'}</FormLabel>
-        <RadioGroup
+      <Box paddingBottom="24px">
+        <RadioGroupField
+          name="additionalInformation.usesAccessibleConvertedVan"
+          label="Is the applicant using an accessible converted van?"
+          required
           value={
-            data.usesAccessibleConvertedVan === null
+            additionalInformation.usesAccessibleConvertedVan === null
               ? undefined
-              : data.usesAccessibleConvertedVan
-              ? '0'
-              : '1'
-          }
-          onChange={value =>
-            onChange({ ...data, usesAccessibleConvertedVan: value === '0' ? true : false })
+              : Number(additionalInformation.usesAccessibleConvertedVan)
           }
         >
           <Stack>
-            <Radio value={'0'}>{'Yes'}</Radio>
-            <Radio value={'1'}>{'No'}</Radio>
+            <Radio value={1} isChecked={additionalInformation.usesAccessibleConvertedVan ?? false}>
```

Comment:
```suggestion
            <Radio value={1} isChecked={!!additionalInformation.usesAccessibleConvertedVan}>
```
This will cast this value to a boolean

Reply:
Actually, is the `isChecked` property necessary here? The `value` prop of the RadioGroup should select the selected option

Reply:
Nope its not, good catch